torch/fx/graph_module.py

```diff
@@ -194,10 +194,20 @@ def __reduce__(self):
     def __deepcopy__(self, memo):
         fake_mod = torch.nn.Module()
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
-        return GraphModule(fake_mod, self.graph)
+        graph_module = GraphModule(fake_mod, self.graph)
+        # skip overwriting generated attributes
+        for attr in ['code', '_graph', '_modules']:
```

Comment:
These fields are not complete. For instance `_parameters` and `_buffers` are not included and copying them might break the nn.Module similar to `_modules`.  Any attempt to list parameters here would be very fragile because changes to `nn.Module` would affect what this list needs to be with no reliable way to inform an editor of nn.Module that this list also needs to be modified.

Reply:
actually I'm not sure why I included `_modules` we probably don't need that. This is just for skipping attributes generated when creating a GraphModule.